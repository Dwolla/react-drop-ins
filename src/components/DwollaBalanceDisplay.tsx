import type { FC } from 'react';
import type { BalanceDisplayProps } from '../types/component-props';

const DwollaBalanceDisplay: FC<BalanceDisplayProps> = (props) => {
  return <dwolla-balance-display {...props} />;
};

export default DwollaBalanceDisplay;
