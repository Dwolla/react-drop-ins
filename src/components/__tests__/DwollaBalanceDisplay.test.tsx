import TestRenderer from 'react-test-renderer';
import DwollaBalanceDisplay from '../DwollaBalanceDisplay';

test('DwollaBalanceDisplay', () => {
  const tree = TestRenderer.create(<DwollaBalanceDisplay customerId="123" />).toJSON();

  expect(tree).toMatchSnapshot();
});

test('with optional props', () => {
  const tree = TestRenderer.create(
    <DwollaBalanceDisplay customerId="123" hideZeroBalance />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
