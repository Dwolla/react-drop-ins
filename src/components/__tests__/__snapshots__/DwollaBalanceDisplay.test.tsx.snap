// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DwollaBalanceDisplay 1`] = `
<dwolla-balance-display
  customerId="123"
/>
`;

exports[`with optional props 1`] = `
<dwolla-balance-display
  customerId="123"
  hideZeroBalance={true}
/>
`;
